torch/nn/quantized/modules/linear.py

```diff
@@ -121,13 +121,36 @@ def weight(self):
     def weight(self, w):
         self._packed_weight = torch.ops.quantized.fbgemm_linear_prepack(w)
 
+    # TODO: These will be duplicated multiple times, should we use multi-inheritance?
```

Comment:
why do we need this stuff as properties in the first place?

Reply:
do we want to enable users to interact with these modules?

Reply:
cc @zafartahirov

Reply:
this is copied from quantized/modules/conv.py btw.

Reply:
I think we probably don't need to allow user interaction with these things, I'll remove.